x-pack/plugins/security_solution/public/detections/components/alerts_table/additional_filters_action/index.test.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
```

Comment:
It would be great to use testing-library tests here like we started to do in other parts of our code: https://github.com/janmonschke/kibana/blob/3943b4cc8751900ba4e60ba506e7d06479e65dfc/x-pack/plugins/security_solution/public/common/components/event_details/insights/simple_alert_table.test.tsx